src/Authenticator/LoginHandler.php

```diff
@@ -284,6 +240,28 @@ public function finishRegistration(HTTPRequest $request)
         return $this->jsonResponse(['success' => true], 201);
     }
 
+    /**
+     * Handle an HTTP request to skip MFA registration
+     *
+     * @param HTTPRequest $request
+     * @return HTTPResponse
+     */
+    public function skipRegistration(HTTPRequest $request)
+    {
+        $schemaGenerator = $this->getSchemaGenerator();
+        if (!$schemaGenerator->canSkipMFA()) {
+            return $this->jsonResponse(
+                ['errors' => [_t(__CLASS__ . '.CANNOT_SKIP', 'You cannot skip MFA registration')]],
+                403
+            );
+        }
+
+        $schemaGenerator->getMember()->update(['HasSkippedMFARegistration' => true])->write();
```

Comment:
I kind of feel like the "schema generator" shouldn't be the service class responsible for providing the "can skip MFA" and the member instance. We may need to split that out into another service class... a "member decision maker service" or something